services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -77,7 +108,11 @@ async def task_create_service_containers(
         await docker_compose_create(shared_store.compose_spec, settings)
 
         progress.publish(message="ensure containers are started", percent=0.95)
-        r = await docker_compose_start(shared_store.compose_spec, settings)
+
+        try:
+            r = await _retry_docker_compose_start(shared_store.compose_spec, settings)
+        except _RetryAgain as e:
+            r = e.command_result
```

Comment:
see? that is exactly what I don't like about the error retrieval in the long running tasks. instead of using well known http codes, with reason string, these are now returned with a string that only the server knows about. and now the client must know it too in order to decipher it.